shoutrrr/cmd/send/send.go

```diff
@@ -46,6 +47,10 @@ func run(cmd *cobra.Command) error {
 	verbose, _ := flags.GetBool("verbose")
 
 	urls, _ := flags.GetStringArray("url")
+	// Remove duplicate service URLs so we don't send to the same
+	// service twice
+	_ = dedupe.Do(&urls)
```

Comment:
```suggestion
	urls = utils.RemoveDuplicates(&urls)
```

I think this should not need an external dependency, especially not one that uses reflection (I am trying to get rid of reflection altogether in `next`).
Since we only need to do this for a string slice, it could be implemented as:

```go
func RemoveDuplicates(src []string) []string {
	unique := make([]string, 0, len(urls))

	for _, s := range src {
		found := false
		for _, us := range unique {
			if us == s {
				found = true
				break
			}
		}
		if !found {
			unique = append(unique, u)
		}
	}

	return unique
}
```

Reply:
You're right. I was probably too focused on making a minimal diff.

I pushed your implementation together with a test for it.